### DISCUSSION TITLE
如何像作者你的博客一样去除banner图

### DISCUSSION THREAD

**Discussion by rumosky** [solstice23/argon-theme#194] [Q&A] [answered]
如题，在后台会有一个很大的banner图，默认内置了几个样式，但设置了背景图之后，感觉banner就很难受。不好看
看到作者你的博客好像banner是透明的，直接默认显示的就是背景图，是如何做到的？

**Comment by rumosky** [comment, marked as answer]
> 如题，在后台会有一个很大的banner图，默认内置了几个样式，但设置了背景图之后，感觉banner就很难受。不好看
> 看到作者你的博客好像banner是透明的，直接默认显示的就是背景图，是如何做到的？

找到了，对不住，仔细看了设置才发现。

**Reply by sbcxk** [reply]
┭┮﹏┭┮，同问在哪

**Comment by yzxoi** [comment]
![image](https://user-images.githubusercontent.com/46484956/120886835-89eb0280-c622-11eb-89f6-4b7bb6bd8389.png)

**Reply by sbcxk** [reply]
好的，谢谢大佬

**Reply by sbcxk** [reply]
![3L506CPI0C2`3$I}N3CA}$1](https://user-images.githubusercontent.com/67060479/120887269-645ef880-c624-11eb-8017-d4439b73c3b4.png)

还想请教一下 演示站的这种日历部分和年度进程怎么设置呀，自己的日历太丑了
![K3_5 T4 {$W}Y5C 4{6 ~GI](https://user-images.githubusercontent.com/67060479/120887266-5e691780-c624-11eb-80ee-5e190b2cb05e.png)
